### PR TITLE
Fix- Catch exception for 404 issue on page load - axios issue

### PR DESCRIPTION
**Pull request by venkatram090993**
*Issue #, if available:*

FiX - Add catch exception for 404 issues on page load - Axios issue

*Description of changes:*

The network request is being sent but we are receiving a 404 response.  Added try catch block to handle the issue,

issue-404

# Checklist

- [ ] Tests have been added and are passing
- [ ] Documentation has been updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/ui/pages/index.tsx [hunk 1]
```diff
@@ -53,19 +53,21 @@ export const TASK_SUB_TABLE_HEADINGS = [
...
-  }
-  
+  const router = useRouter();
+  const programsContext = useProgramsContext();
+  const isFirstTimeVisitor = !programsContext.programs.length;
+
+  useEffect(() => {
+    if (isFirstTimeVisitor) {
+      router.push('/workspace/stepform');
+      return null;
```
Comment (proserve) [inline packages/ui/pages/index.tsx:67]: @venkatram090993 I think redirecting to `/workspace/stepform` is enough no need to return `null`. 
The `useEffect` hook returns only functions which will be used later for cleanup when the component is Unmounting.

**Comment by proserve** [comment]
@venkatram090993 some formatting changes in existent code are there, have we changed `Eslint` or `Prettier` config

**Comment by brettstack** [comment]
Let's show an error message instead of just logging to console.

**Review by brettstack — APPROVED** [review]

**Pull request merged**